oshi-core/src/main/java/oshi/software/os/linux/LinuxInstalledApps.java

```diff
@@ -94,10 +96,10 @@ private static List<ApplicationInfo> parseLinuxAppInfo(List<String> output) {
                         ParseUtil.parseLongOrDefault(parts[4], 0L), // Date Epoch
                         additionalInfo);
 
-                appInfoList.add(app);
+                appInfoSet.add(app);
             }
         }
 
-        return appInfoList;
+        return appInfoSet;
```

Comment:
After changing return types, create a (order preserving) list from the (insertion order preserving) set.
```suggestion
        return new ArrayList<>(appInfoSet);
```

Reply:
Addressed in the latest commit